SpywareFilter/sections/tracking_servers.txt

```diff
@@ -38,6 +38,7 @@
 !||i.snssdk.com/log/*
 !||i.snssdk.com/slardar/sdk.js
 !
+||110.93.143.144:8080^
```

Comment:
Unfortunately, I don't see such this request.

Reply:
It looks strange. I can see that request.
Please search `logcollect.hci` at Network section of Developer Tools.

Reply:
I can see it now. `||110.93.143.144^$third-party` is ok here.

Reply:
48e5f2f50956d21bf8f34bf704233433b18ff30e